### PR TITLE
MM-54323 - Calls: Incoming call from different server

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- NOTE: this is the same as https://github.com/mattermost/mattermost-mobile/pull/7624 , but that PR was closed when I rebased onto the new branch from https://github.com/mattermost/mattermost-mobile/pull/7620 (I think I know what happened). Anyway, apologies for the repeated request for review.  
- The new commit is: https://github.com/mattermost/mattermost-mobile/pull/7646/commits/0ac5662ca647cf55bdd91f782900d2bad79a50a0 which adds the server name to the incoming call notification whenever the user has >1 active server (as @streamer45 and @matthewbirtch suggested in https://github.com/mattermost/mattermost-mobile/pull/7624#issuecomment-1785784525 ).

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-54323

#### Checklist
- [ ] Added or updated unit tests (required for all new features)
- [x] Has UI changes
- [ ] Includes text changes and localization file updates
- [x] Have tested against the 5 core themes to ensure consistency between them.

#### Device Information
- Android: 13, Pixel 6
- Android: 13, Galaxy Tab s7+
- iOS: 16.5.1, iPhone 14

#### Screenshots

| iOS other server DM | iOS other server GM | Android tablet other server GM |
|--|--|--|
| IMG_ECF9D61F6BAC-1 | IMG_C508AF38F42A-1 | Screenshot_20231020_142027_Mattermost Beta |

#### Release Note

```release-note
Calls: Added the server name to the call notification banner for a better user experience.
```

### CODE REVIEW

File: app/products/calls/hooks.ts [hunk 1]
```diff
@@ -108,3 +109,27 @@ export const usePermissionsChecker = (micPermissionsGranted: boolean) => {
         }
     }, [appState]);
 };
+
+export const useCallsAdjustment = (serverUrl: string, channelId: string) => {
```
Comment (streamer45) [inline app/products/calls/hooks.ts:113]: Isn't this from the other PR?

**Review by streamer45 — APPROVED** [review]
Thanks Chris!

**Review by enahum — APPROVED** [review]
Lgtm but did not do a detailed review

File: app/products/calls/hooks.ts (hunk 1 above)
Comment (cpoile) [reply app/products/calls/hooks.ts:113]: Yes, I'm not sure why it's showing a diff here, it's on main. 🤷

**Comment by cpoile** [comment]
@matthewbirtch Does this look good?

**Comment by matthewbirtch** [comment]
It's looking good @cpoile. The only thing I can see is that when other banners are showing, the margin on the bottom is missing so the banner is sitting right up against the banner below. This doesn't seem to be the case when we're not showing the server name. Once we update that, we should be good. 
IMG_1479

**Comment by cpoile** [comment]
@matthewbirtch Thanks, fixed!

**Comment by cpoile** [comment]
@matthewbirtch ping on this, thanks!

**Review by matthewbirtch — APPROVED** [review]
Sorry for the delay, looks good!

**Pull request merged**